BusScheduleDB/1 tables.sql

```diff
@@ -0,0 +1,17 @@
+use BusScheduleDB
+go
+drop table if exists StudentBusSchedule
```

Comment:
table name in drop different than create

Reply:
Fixed, first named it StudentBusSchedule and then changed it.